src/components/TimeseriesChartExport/TimeseriesChartExport.tsx

```diff
@@ -0,0 +1,417 @@
+import { Aggregate, DatapointsMultiQuery, InternalId } from '@cognite/sdk';
+import { TimeSeries } from '@cognite/sdk/dist/src/resources/classes/timeSeries';
+import { TimeSeriesList } from '@cognite/sdk/dist/src/resources/classes/timeSeriesList';
+import {
+  Alert,
+  Button,
+  Checkbox,
+  DatePicker,
+  Form,
+  Input,
+  Modal,
+  Radio,
+} from 'antd';
+import { RangePickerValue } from 'antd/lib/date-picker/interface';
+import { FormComponentProps } from 'antd/lib/form';
+import { ColProps } from 'antd/lib/grid';
+import moment, { Moment } from 'moment-timezone';
+import React, { SyntheticEvent, useContext, useEffect, useState } from 'react';
+import { ClientSDKContext } from '../../context/clientSDKContext';
+import { withDefaultTheme } from '../../hoc/withDefaultTheme';
+import { AnyIfEmpty, PureObject } from '../../interfaces';
+import { defaultTheme } from '../../theme/defaultTheme';
+import { datapointsToCSV, Delimiters, downloadCSV } from '../../utils/csv';
+import { getGranularityInMS } from '../../utils/utils';
+import { ComplexString } from '../common/ComplexString/ComplexString';
+
+export type FetchCSVCall = (
+  request: DatapointsMultiQuery,
+  opts: CsvParseOptions
+) => Promise<string>;
+
+export type FetchTimeseriesCall = (
+  ids: InternalId[]
+) => Promise<TimeSeriesList>;
+
+export interface CsvParseOptions {
+  aggregate: Aggregate;
+  delimiter: Delimiters;
+  readableDate: boolean;
+  granularity: string;
+}
+
+export interface FormItemLayout {
+  labelCol: ColProps;
+  wrapperCol: ColProps;
+}
+
+export interface ChartExportFormFields {
+  range: Moment[];
+  granularity: string;
+  delimiter: Delimiters;
+  readableDate: boolean;
+}
+
+export interface TimeseriesChartExportProps extends FormComponentProps {
+  timeseriesIds: InternalId[];
```

Comment:
sorry didn't see this one
could you please use just `number[]` like we do in other components? then the user doesn't have to use objects